sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -355,29 +355,33 @@ object SQLConf {
 
   val SHUFFLE_TARGET_POSTSHUFFLE_INPUT_SIZE =
     buildConf("spark.sql.adaptive.shuffle.targetPostShuffleInputSize")
-      .doc("The target post-shuffle input size in bytes of a task.")
+      .doc("The target post-shuffle input size in bytes of a task. This configuration only has " +
+        s"an effect when '${ADAPTIVE_EXECUTION_ENABLED.key}' is enabled.")
```

Comment:
and also when `spark.sql.adaptive.shuffle.reducePostShufflePartitions.enabled` is true?

Reply:
Thanks, fix the doc of SHUFFLE_MIN_NUM_POSTSHUFFLE_PARTITIONS and SHUFFLE_TARGET_POSTSHUFFLE_INPUT_SIZE.